### PR TITLE
Chore: reuse AVPro component

### PR DESCRIPTION
**Pull request by dalkia**
# Pull Request Description

### Prerequisites
- [ ] List any required setup steps
- [ ] Include environment/configuration requirements

### Test Steps
1. First step
2. Second step
3. Expected result after step 2
4. ...

### Additional Testing Notes
- Note any edge cases to verify
- Mention specific areas that need careful testing
- List known limitations or potential issues

## Quality Checklist
- [ ] Changes have been tested locally
- [ ] Documentation has been updated (if required)
- [ ] Performance impact has been considered
- [ ] For SDK features: Test scene is included

## Code Review Reference
Please review our [Code Review Standards](https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md) before submitting.

### CODE REVIEW

File: Explorer/Assets/DCL/SDKComponents/MediaStream/Wrapper/MediaPlayerPluginWrapper.cs [hunk 1]
```diff
@@ -11,10 +11,11 @@
 using System.Collections.Generic;
 using UnityEngine;
 using RenderHeads.Media.AVProVideo;
-using DCL.ECSComponents;
 using DCL.FeatureFlags;
 using DCL.SDKComponents.MediaStream.Settings;
 using DCL.Settings;
+using System.Linq;
```
Comment (m3taphysics) [inline Explorer/Assets/DCL/SDKComponents/MediaStream/Wrapper/MediaPlayerPluginWrapper.cs]: Where is LINQ?

Comment (dalkia) [reply Explorer/Assets/DCL/SDKComponents/MediaStream/Wrapper/MediaPlayerPluginWrapper.cs]: Silly leftover from other ideas. Removing

File: Explorer/Assets/DCL/SDKComponents/MediaStream/MediaPlayerCustomPool.cs [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+            return mediaPlayer;
+        }
+
+        private async UniTask TryUnloadAsync()
+        {
+            while (true)
+            {
+                //We will do this analysis every two minute
+                await UniTask.Delay(TimeSpan.FromMinutes(2));
```
Comment (popuz) [inline Explorer/Assets/DCL/SDKComponents/MediaStream/MediaPlayerCustomPool.cs:54]: so let's extract magic 2 to a constant?

**Review by popuz — APPROVED** [review]
looks good 👍 2  small clean-up remarks

File: Explorer/Assets/DCL/SDKComponents/MediaStream/MediaPlayerCustomPool.cs (hunk 2 above)
Comment (popuz) [reply Explorer/Assets/DCL/SDKComponents/MediaStream/MediaPlayerCustomPool.cs:54]: and another 300 too

**Review by m3taphysics — APPROVED** [review]
